docs/internal/docs.rst

```diff
@@ -193,8 +217,229 @@ omgissue
+
+.. note::
+
+  If using nested lists, reStructuredText requires these to have empty lines before and after as shown in the example above.
+
+Content must be contained somewhere within a section, indicated by ``news-start-section`` and ``news-end-section``.
+Having text outside sections that isn't a comment or a special news directive is an error.
+
+Content should provide a reasonable amount of context to users so they can figure out if and how a change will impact or benefit them.
+Some of this can be achieved through :ref:`subsections <docs-news-subsections>`.
+Content should link to the rest of the Sphinx documentation and to source files using ``:ghfile:`` if relevant.
+
+PRs will usually have just have one fragment file, but can have more then one.
+For example, say there is a PR that is making changes associated with an existing fragment file, but also has changes that are separate from the PRs in that fragment.
+This might indicate a new PR should be made for those changes, but if that's impractical, a new separate fragment file for those changes can be made.
+
+Directives
+----------
+
+news-prs
+^^^^^^^^
+
+``news-prs`` should declare all the PRs that make up the changes to OpenDDS described in the fragment.
+It's an error to omit ``news-prs`` from a fragment or to have more than one ``news-prs`` directive.
+Do not add ``#`` at the start of the PR numbers.
+Please add follow-on PRs separated by spaces as they are created.
+
+.. code-block:: rst
+
+  .. news-prs: 1002 1003 1008
+
+If the changes don't have a PR associated with them, such as a policy change, then put ``none`` instead:
+
+.. code-block:: rst
+
+  .. news-prs: none
+
+news-start-section and news-end-section
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``news-start-section`` and ``news-end-section`` directives contain the news content and define how they're grouped.
+All sections with the same name across all the fragments are merged together in the final result.
+The top-level sections must be one of the following:
+
+``Additions``
+  New user-relevant features
+
+``Platform Support and Dependencies``
+  Additional support or changes to support for OS, C++ standards, compilers, and dependencies.
+
+``Deprecations``
+  User-relevant features that are being planned on being removed, but removing now would break a significant number of user's use cases.
+  This marks them for removal in the next major version.
+
+``Removals``
+  User-relevant features removed that were either deprecated, "experimental", or "internal" that users might be relying on anyways.
+
+``Security``
+  Fixes for issues that might compromise the security of a system using OpenDDS.
+  This doesn't need to be related to :doc:`DDS Security </devguide/dds_security>`.
+
+``Fixes``
+  User-relevant fixes not related to security or documentation.
+
+``Documentation``
+  Signifiant changes to documentation, either in the primary Sphinx documentation or in secondary documentation.
```

Comment:
```suggestion
  Significant changes to documentation, either in the primary Sphinx documentation or in secondary documentation.
```

Reply:
It's annoying that's a word. Either I must have either typed that as is or mispelled significant enough that it suggested "signifiant" first and I wasn't paying attention.